src/test/kotlin/com/atlassian/performance/tools/awsinfrastructure/AwsCliIT.kt

```diff
@@ -0,0 +1,31 @@
+package com.atlassian.performance.tools.awsinfrastructure
+
+import com.atlassian.performance.tools.sshubuntu.api.SshUbuntuContainer
+import org.assertj.core.api.Assertions
+import org.junit.Test
+import java.util.concurrent.Executors
+import java.util.concurrent.TimeUnit
+
+class AwsCliIT {
+
+    @Test
+    fun shouldBeParallelOnDifferentSshConnections() {
+        val executor = Executors.newCachedThreadPool()
```

Comment:
It's not shut down, but I wonder if it's worth any complexity, because leaking threads shouldn't be a problem in a throwaway test runtime. The integration tests are not a long-running service.